plugin/gcp_config.go

```diff
@@ -21,6 +20,15 @@ type gcpConfig struct {
 	IAMAuthMetadata *authmetadata.Handler   `json:"iam_auth_metadata_handler"`
 	GCEAliasType    string                  `json:"gce_alias"`
 	GCEAuthMetadata *authmetadata.Handler   `json:"gce_auth_metadata_handler"`
+
+	// APICustomEndpoint overrides the service endpoint for www.googleapis.com
+	APICustomEndpoint string `json:"api_custom_endpoint"`
```

Comment:
Not sure on name here. There isn't a service I can pinpoint that maps directly to the `www` subdomain. This is used to override the default service endpoints in [go-gcp-common](https://github.com/hashicorp/go-gcp-common/blob/master/gcputil/credentials.go#L31).